README.md

```diff
@@ -1 +1,144 @@
-# promise-utils
+# promise-utils [![ContinuousBuild](https://github.com/api3dao/promise-utils/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/api3dao/promise-utils/actions/workflows/main.yml)
+
+> A simple package for a functional and typesafe error handling
+
+## Installation
+
+To install this package run either:
+
+`yarn add @api3/promise-utils`
+
+or if you use npm
+
+`npm install @api3/promise-utils --save`
+
+## Usage
+
+The API is small and well focused on the error handling problem. The main functions of this package are `go` and
+`goSync` functions. They accept a function to execute. If the function executes without an error, a success response
```

Comment:
That is fragile since the code often moves if the link points to master. If you point to a specific commit hash it gets outdated.

I, as a developer, would not appreciate it :D

Reply:
![touche](https://c.tenor.com/PY4uJ0NkywQAAAAM/the-goon-dodgeball.gif)